### PR TITLE
Set FINOS badge to “Released” ahead of 1.0 release

### PR DESCRIPTION
**Pull request by rikoe**
Relating to #32 release checklist for 1.0 release.

Also fix StackOverflow tag in badge

### CODE REVIEW

**Comment by finos-admin** [comment]
Folks,

I believe there is some confusion here and understandly why. Let me try to elaborate:

- The "released" lifecycle status is not simply achieved by "releasing" the 1.0.0 of the the code. In fact this status was previous called "active" and represents a badge of quality for the project. For the record I was strongly opposed to rename this state to released when the (Ex-SSF) Esco proposed that exactly to avoid this type of confusion (/cc @ftbb)

- As per our lifecycle (https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530756/Project+and+Working+Group+Lifecycles), the "released" state is achieved by the PMC voting to trigger "activation" of a project (i.e. the transition that brings the project to "released", formerly "active" state). In order to do so, the project must check all the boxes for "activation" -> see here https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation.

- In other words having a release out is a mandatory but not sufficient to be in "released" state (again I know it's EXTREMELY confusing, but I am just the messenger here). See penultimate criteria of the activation https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation.

So FDC3, until the PMC takes a vote to "activate" (because all the boxes in https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation are checked), is in state "Incubating".

I am very supportive of this vote happening ASAP (seems FDC3 would check all the boxes) but until then, can you please re-open this issue and re-instate the "incubating" badge?

**Comment by mindthegab** [comment]
Folks (@nkolba @rikoe)

I believe there is some confusion here and understandly why. Let me try to elaborate:

The "released" lifecycle status is not simply achieved by "releasing" the 1.0.0 of the the code. In fact this status was previous called "active" and represents a badge of quality for the project. For the record I was strongly opposed to rename this state to released when the (Ex-SSF) Esco proposed that exactly to avoid this type of confusion (/cc @ftbb)

As per our lifecycle (https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530756/Project+and+Working+Group+Lifecycles), the "released" state is achieved by the PMC voting to trigger "activation" of a project (i.e. the transition that brings the project to "released", formerly "active" state). In order to do so, the project must check all the boxes for "activation" -> see here https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation.

In other words having a release out is a mandatory but not sufficient to be in "released" state (again I know it's EXTREMELY confusing, but I am just the messenger here). See penultimate criteria of the activation https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation.

So FDC3, until the PMC takes a vote to "activate" (because all the boxes in https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530376/Activation are checked), is in state "Incubating".

I am very supportive of this vote happening ASAP (seems FDC3 would check all the boxes) but until then, can you please re-open this issue and re-instate the "incubating" badge?

**Comment by mindthegab** [comment]
As per our convo on email today, @nkolba and @rikoe don't consider this a blocker for 1.0.0. 

I or @brooklynrob will make sure to attend the next PMC meeting and explain the havoc.

I apologize for the confusion created by the state called "Released" (note the upper case) which clearly makes you think that once that you have "released" (note the lower case) the software then you qualify. Definitely not your fault and something we need to review.
